demos/dermatology.html

```diff
@@ -375,7 +379,7 @@
                 ;
                 if (first) {

```

Comment:
This is inside the resize method - and the last line of code calls ``` frame.doResize();``` so on the "first call" we want to process the url, but never again.